Readme.md

```diff
@@ -11,15 +11,21 @@ Simple flavour of node/iojs binary management, no subshells, no profile setup, n
 
 ## Installation
 
+With node/iojs already installed:
+
     $ npm install -g n
 
-or
+or, by cloning this repo and running:
 
-    $ make install
+    $ PREFIX=$HOME make install
 
-to `$HOME`. Prefix later calls to `n` with `N_PREFIX=$HOME`
+to install to `$HOME`. Prefix later calls to `n` with `N_PREFIX=$HOME` or export `N_PREFIX` in your shell initialization file.
```

Comment:
Shouldn't we instead suggest installing n with just `$ make install`, but note that you can adjust the path for installation? Also, when installing n while passing the PREFIX variable to the Makefile, the $N_PREFIX won't get adjusted. The PREFIX variable is the path where n will download Node.js/io.js, the $N_PREFIX variable is the path where Node.js/io.js will be installed into.